osu.Game/Screens/Play/PlayerLoader.cs

```diff
@@ -32,7 +33,9 @@ public class PlayerLoader : ScreenWithBeatmapBackground
 
         private Player player;
 
-        private Container content;
+        private FacadeContainer facadeContainer;
+
+        protected virtual FacadeContainer CreateFacadeContainer() => new FacadeContainer();
```

Comment:
let's not do this

Reply:
going to go ahead and delete the PlayerLoader test in TestCaseLogoFacadeContainer then, since we can already test it in TestCasePlayerLoader